src/ark/phenotyping/pixie_preprocessing.py

```diff
@@ -6,6 +6,7 @@
 import numpy as np
 import pandas as pd
 import scipy.ndimage as ndimage
+from natsort import natsorted
```

Comment:
Just curious - what is this for?

Reply:
Oh I was planning on using it to sort the fov names, but I didn't end up needing it. I'll remove it!